### ISSUE TITLE
Remove KituraRequest

### ISSUE THREAD

**Issue by ianpartridge** [RuntimeTools/SwiftMetrics#165]
`KituraRequest` has been deprecated and moved to https://github.com/IBM-Swift-Sunset/Kitura-Request

We should remove it from SwiftMetrics.

**Comment by sjanuary** [comment]
@ianpartridge is there a timescale for KituraRequest being deleted entirely? E.g. are we ok if we delay doing this for a couple of months or is it time critical?

**Comment by ianpartridge** [comment]
It's probably going to sit on GitHub indefinitely.

However we should get removing it from SwiftMetrics into plan because SwiftMetrics is built into our Kitura on-ramp (`kitura init`) and hence contributes a lot to this:

```
$ time kitura init
...
real	1m12.561s
user	2m36.356s
sys	0m28.369s
$
```

😢

**Comment by sjanuary** [comment]
Fixed by #172
